src/libp2p/Libp2p.Core.TestsBase/TestChannel.cs

```diff
@@ -1,6 +1,7 @@
 // SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
 // SPDX-License-Identifier: MIT
 
+using Newtonsoft.Json.Linq;
```

Comment:
Do we really need this?

Reply:
Nope, removed with some other unused usings